AsBuiltReport.Fortinet.FortiGate.json

```diff
@@ -1,6 +1,9 @@
 {
     "Report": {
         "Name": "Fortinet FortiGate As Built Report",
+        "Assets": "Your Firewall HostNames",
+        "ProjectRef": "Your Project Reference name and number",
+        "ClientName": "Your Client's Name",
```

Comment:
Client/Customer name is already defined in the AsBuiltReport Core configuration JSON file.

Reply:
@tpcarman, there's a usecase for having two sets of company details, one for service provider and other for client. What do you think?

Reply:
This sounds like a specific use case, not a generic one which would be needed by all users? If you want to insert specific details into your reports you should be creating a custom style script.